src/components/Calculator.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <div className="calculate-place">0</div>
+        <div className="functions-place">
+
+          <div className="rows">
+            <button type="button" className="btn-gray">AC</button>
+            <button type="button" className="btn-gray">+/-</button>
+            <button type="button" className="btn-gray">%</button>
+            <button type="button" className="btn-orang">+</button>
+          </div>
+
+          <div className="rows">
+            <button type="button" className="btn-gray">7</button>
+            <button type="button" className="btn-gray">8</button>
+            <button type="button" className="btn-gray">9</button>
+            <button type="button" className="btn-orang">x</button>
+          </div>
+          <div className="rows">
+            <button type="button" className="btn-gray">4</button>
+            <button type="button" className="btn-gray">5</button>
+            <button type="button" className="btn-gray">6</button>
+            <button type="button" className="btn-orang">-</button>
+          </div>
+          <div className="rows">
+            <button type="button" className="btn-gray">1</button>
+            <button type="button" className="btn-gray">2</button>
+            <button type="button" className="btn-gray">3</button>
+            <button type="button" className="btn-orang">+</button>
+          </div>
+          <div className="rows">
+            <button type="button" className="btn-gray">AC</button>
+            <button type="button" className="btn-gray">+/-</button>
+            <button type="button" className="btn-gray">%</button>
+            <button type="button" className="btn-orang">+</button>
+          </div>
```

Comment:
- [ ] The symbol on line 19 should be the divide operation sign, not plus. Kindly fix that using the correct sign (&divide;) or the HTML code: `&divide;`

- [ ] The code from lines 41 to 44 is a repetition of lines 16 to 19. Kindly remove them.

  Your layout | Expected layout
  :---:|:---:
  ![dkddii](https://user-images.githubusercontent.com/81468675/207291754-7384bd75-3736-492b-a685-a38f0143a013.jpg) | ![rurt](https://user-images.githubusercontent.com/81468675/207291750-8e4caaa7-a32e-435f-94e2-51ceac2192e8.jpg)